docs/src/pages/getting-started/solid.mdx

```diff
@@ -293,42 +293,43 @@ export default function Home() {
 }
 ```
 
-## Advanced usage: `useUploadThing`
+## Advanced usage: `createUploadThing`
 
 For advanced use cases, the premade components might not be flexible enough to
 suit your needs. In that case, you can use the `useUploadThing` hook to build
 your own components with a custom upload flow:
 
 <Steps>
 
-### Create the hook
+### Create the helper
```

Comment:
idk if there is an established name for these `createX` functions in Solid...

Reply:
I think it would be considered a "store" using Solid's vocabulary?
![image](https://github.com/pingdotgg/uploadthing/assets/11494384/4f635c8c-f6b9-4a25-bd73-9a874bd35f80)
<https://docs.solidjs.com/advanced-concepts/fine-grained-reactivity#reactive-primitives>

Reply:
Not actually sure there is one as I look into this more

Reply:
`createX` functions are for reactive utilies/helper/primitives[^1], so I would say that calling it a helper would be correct.

[^1]: <https://primitives.solidjs.community/#make-non-reactive-vs-create-reactive>